documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -48,6 +48,15 @@ The following rules shall be adhered to when serializing a submodel with the Ser
 
 * Submodel elements defined in the submodel other than the ones mentioned above are not subject to serialization of that SerializationModifier.
 
+
+
+The following rules shall be adhered to when serializing a submodel element different than a submodel with the SerializationModifier _Value_:
+
+* The submodel element is serialized as an unnamed JSON object.
+* A submodel element is considered a leaf submodel element if it does not contain other submodel elements. A leaf submodel element follows the rules for the different submodel elements considered in the serialization, as described for a submodel above. If it is not a leaf element, the serialization rules must be transitively followed until the value is a leaf submodel element.
+
+Optional submodel elements are omitted if not specified for the requesting submodel or submodel element.
```

Comment:
What does that even mean? Either they are part of the serialization or not. What would be an example for this rule to apply that is not covered by other rules/definitions?